utils/src/ooinstall/ansible_plugins/facts_callback.py

```diff
@@ -27,9 +27,19 @@ def on_any(self, *args, **kwargs):
     def runner_on_failed(self, host, res, ignore_errors=False):
         pass
 
```

Comment:
pylint is unhappy with with why things are being accessed. Until we can get a better fix though, I suggest we just add `#pylint: disable=protected-access` and get this merged in to unblock things.